gammapy/maps/hpx/geom.py

```diff
@@ -31,34 +31,34 @@
 
 
 class HpxGeom(Geom):
-    """Geometry class for HEALPIX maps.
+    """Geometry class for HEALPix maps.
 
     This class performs mapping between partial-sky indices (pixel
-    number within a HEALPIX region) and all-sky indices (pixel number
-    within an all-sky HEALPIX map).  Multi-band HEALPIX geometries use
+    number within a HEALPix region) and all-sky indices (pixel number
+    within an all-sky HEALPix map).  Multi-band HEALPix geometries use
     a global indexing scheme that assigns a unique pixel number based
     on the all-sky index and band index.  In the single-band case the
-    global index is the same as the HEALPIX index.
+    global index is the same as the HEALPix index.
 
-    By default the constructor will return an all-sky map.
+    By default, the constructor will return an all-sky map.
     Partial-sky maps can be defined with the ``region`` argument.
 
     Parameters
     ----------
     nside : `~numpy.ndarray`
-        HEALPIX nside parameter, the total number of pixels is
-        12*nside*nside.  For multi-dimensional maps one can pass
-        either a single nside value or a vector of nside values
-        defining the pixel size for each image plane.  If nside is not
+        HEALPix NSIDE parameter, the total number of pixels is
+        12*nside*nside. For multi-dimensional maps one can pass
+        either a single ``nside`` value or a vector of ``nside`` values
+        defining the pixel size for each image plane. If ``nside`` is not
         a scalar then its dimensionality should match that of the
-        non-spatial axes. If nest is True, nside must be a power of 2,
+        non-spatial axes. If nest is True, ``nside`` must be a power of 2,
         less than 2**30.
     nest : bool
-        True -> 'NESTED', False -> 'RING' indexing scheme
-    frame : str
-        Coordinate system, "icrs" | "galactic"
+        Indexing scheme. If True, "NESTED" scheme. If False, "RING" scheme.
+    frame : {"icrs", "galactic"}
+        Coordinate system.
     region : str or tuple
-        Spatial geometry for partial-sky maps.  If none the map will
+        Spatial geometry for partial-sky maps.  If None, the map will
```

Comment:
Again, a lot of additional space in these files.
```suggestion
        Spatial geometry for partial-sky maps. If None, the map will
```